### DISCUSSION TITLE
Simple Icons

### DISCUSSION THREAD

**Discussion by noraj** [ryanoasis/nerd-fonts#818] [Ideas]
Idea : adding Simple Icons

- https://simpleicons.org
- https://github.com/simple-icons/simple-icons

**Comment by seagle0128** [comment]
This should address #342 as well.

**Comment by Finii** [comment]
As far as I can see `simple-icons` does not have any glyph-to-codepoint translation.
They have a complete table of all glyph names (`slugs.md`).
(Both is neither positive nor negative.)

There are over 2,200 icons in the set. **The question is where to put them?** Probably in PUA-A.
That is always the most important question for any added glyph sets.

See also
* https://github.com/ryanoasis/nerd-fonts/issues/173#issuecomment-1102379463
* https://github.com/ryanoasis/nerd-fonts/wiki/Glyph-Sets-and-Code-Points#overview

**Comment by Aman9das** [comment]
https://github.com/simple-icons/simple-icons-font

They have a font repo, linking it here..
